.gitattributes

```diff
@@ -0,0 +1 @@
+*.lockb binary diff=lockb
```

Comment:
bad rebase please rebase the code

Reply:
I rollbacked rebase
Propose to merge this. In new PR will be only css and images in css.